pkg/koordlet/util/system/kidled_util.go

```diff
@@ -132,13 +132,32 @@ func (i *ColdPageInfoByKidled) GetColdPageTotalBytes() uint64 {
 	sum := func(nums ...[]uint64) uint64 {
 		var total uint64
 		for _, v := range nums {
-			for _, num := range v {
-				total += num
+			// [1,2)  [2,5)  [5,15)	 [15,30)  [30,60)  [60,120)  [120,240) [240,+inf)
+			for i, num := range v {
+				if i > 2 {
```

Comment:
If the index `2` is an uncertain threshold, it should be parameterized. Same below.